docs/pre-release/reference/vpn.md

```diff
@@ -0,0 +1,157 @@
+ * How is your VPN pushing DNS configuration? It may be useful to add the contents of /etc/resolv.conf
+```
+
+#### Interpreting test results
+
+##### Case 1: VPN masked by cluster
+
+In this case, the tool will inform you that a pod or service subnet is masking a CIDR that the VPN
+routes:
+
+```
+❌ pod subnet 10.0.0.0/19 is masking VPN-routed CIDR 10.0.0.0/16. This usually means Telepresence will be able to connect to your cluster, but hosts on your VPN may be inaccessible while telepresence is connected; to resolve:
+        * Move pod subnet 10.0.0.0/19 to a subnet not mapped by the VPN
+                * If this is not possible, ensure that any hosts in CIDR 10.0.0.0/16 are placed in the never-proxy list
+```
+
+Typically, this means that all VPN hosts within `10.0.0.0/19` will be rendered inaccessible while
+telepresence is connected.
+
+The ideal resolution in this case is to simply move the pods to a different subnet. This is possible,
+for example, in EKS by configuring a [new CIDR range](https://aws.amazon.com/premiumsupport/knowledge-center/eks-multiple-cidr-ranges/) for the pods.
+In this case configuring the pods to be located in `10.1.0.0/19` would clear the VPN and allow you
+to reach hosts inside the VPC's `10.0.0.0/19`
+
+However, it is not always easy, or possible, to move the pods to a different subnet.
+In these cases, you should use the [never-proxy](./config#neverproxy) configuration to prevent certain
+hosts from being masked.
+This might be particularly important for DNS resolution. In an AWS ClientVPN VPN it is often
+customary to set the `.2` host as a DNS server (e.g. `10.0.0.2` in this case):
+
+<img src="../../images/vpn-dns.png" />
+
+If this is the case for your VPN, you should place the DNS server in the never-proxy list for your
+cluster. In your kubeconfig file, simply add a `telepresence` extension like so:
+
+```yaml
+- cluster:
+    server: https://127.0.0.1
+    extensions:
+    - name: telepresence.io
+      extension:
+        never-proxy:
+        - 10.0.0.2/32
+```
+
+##### Case 2: Cluster masked by VPN
+
+In this case, the tool will inform you that a pod or service subnet is being masked by a CIDR
+that the VPN routes:
+
+```
+❌ pod subnet 10.0.0.0/8 being masked by VPN-routed CIDR 10.0.0.0/16. This usually means that Telepresence will not be able to connect to your cluster. To resolve:
+        * Move pod subnet 10.0.0.0/8 to a subnet not mapped by the VPN
+                * If this is not possible, consider shrinking the mask of the 10.0.0.0/16 CIDR (e.g. from /16 to /8), or disabling split-tunneling
+```
+
+Typically this means that pods within `10.0.0.0/8` will not be accessible while the VPN is
+connected.
+
+As with the first case, the ideal resolution is to move the pods away, but this may not always
+be possible. In that case, your best bet is to attempt to shrink the VPN's CIDR
+(i.e. make it route more hosts) to make Telepresence's routes win by virtue of specificity.
```

Comment:
Avoid latinisms; remove "i.e."

Reply:
Is there a better way to specify that something is just an example then? Or should we explicitly say, "For example," or something similar?

Reply:
`i.e.` means "that is" so maybe we should just phrase it as `that is, make it route more hosts`